src/main/resources/assets/minecolonies/gui/citizen/family.xml

```diff
@@ -1,27 +1,29 @@
-<window size="210 244"  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/ldtteam/BlockUI/version/1.20.1/src/main/resources/assets/blockui/gui/block_ui.xsd">
+<window
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/ldtteam/BlockUI/version/1.20.1/src/main/resources/assets/blockui/gui/block_ui.xsd"
+    size="210 244">
     <layout source="minecolonies:gui/citizen/nav.xml"/>
 
-    <text pos="80 15" size="110 11" color="black" label="§n$(com.minecolonies.coremod.gui.citizen.family.title):§r"/>
-    
-    <text size="110 11" pos="90 40" color="black"  label="§n$(com.minecolonies.coremod.gui.citizen.family.parents):§r"/>
-    <text id="parentA" size="130 11" pos="50 51" color="black" textalign="MIDDLE"/>
-    <text id="parentB" size="130 11" pos="50 62" color="black" textalign="MIDDLE"/>
+    <text color="black" label="§n$(com.minecolonies.coremod.gui.citizen.family.title):§r" pos="80 15" size="110 11"/>
 
-    <text size="110 11" pos="90 80" color="black"  label="§n$(com.minecolonies.coremod.gui.citizen.family.partner):§r"/>
-    <text id="partner" size="130 11" pos="50 95" color="black" textalign="MIDDLE"/>
+    <text color="black" label="§n$(com.minecolonies.coremod.gui.citizen.family.parents):§r" pos="90 40" size="110 11"/>
+    <text color="black" id="parentA" pos="50 51" size="130 11" textalign="MIDDLE"/>
```

Comment:
can we have more logical sorting for these? like id, pos, size going first

Reply:
yeah, Id like id/pos/size by alphabet

Reply:
@Raycoms @Nightenom Is this something we could merge without this change, the linked pull request I made on the prettier plugin has been denied. So I'm not going to be able to force these properties to appear at the front.

Keep it as it is now or scrap the idea?

Reply:
Could we use the intellij format rules to better format those things locally just, instead of relying on plugins?